geometry_msgs/PoseStamped[] initWayPoints
---
bool success